fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/KeycloakFraction.java

```diff
@@ -27,7 +28,10 @@
  */
 @WildFlyExtension(module = "org.keycloak.keycloak-adapter-subsystem")
 @WildFlySubsystem("keycloak")
-@DeploymentModule(name = "org.keycloak.keycloak-core")
+@DeploymentModules({
+  @DeploymentModule(name = "org.keycloak.keycloak-core"),
+  @DeploymentModule(name = "org.wildfly.swarm.keycloak", slot = "deployment", export = true, metaInf = DeploymentModule.MetaInfDisposition.IMPORT)
+})
```

Comment:
Isn't the `@DeploymentModule` annotation `@Repeatable`?

Reply:
Thanks, did not know that, updating...